### PR TITLE
Update for Tailwind ver 1

### PR DESCRIPTION
**Pull request by rverrips**
Hi Jeffery
Not sure if this is the best approach (certainly the simplest) by just updating to run with Tailwind-ver 1.   Could potentially do a fs.stat and see which of the config files is there, or simply version the npm  accordingly to match tailwind version numbers?
Either way, let me know if you'd like me to help with this in any other ways.
Appreciate all you do!
Roy

### CODE REVIEW

**Comment by johannebert** [comment]
Hey,

you can just paste a right config filename into function like this:

```js
mix.tailwind('./tailwind.config.js');
```

Regards

**Comment by rverrips** [comment]
Thanks @johannebert - That is what I've been doing, but would suggest it's better to have the defaults set as per the currently supported upstream which will soon be v1 which is why I suggested to version the npm according and I included the peer dependency sample.

So have version < 1.0.0 use editing and >= 1.0.0 use the suggested change.     

Either way, just trying to help where I can - It's a trivial change but has potential to affect a large number of people ...

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
   },
   "homepage": "https://github.com/jeffreyway/laravel-mix-tailwind#readme",
   "peerDependencies": {
-    "laravel-mix": "^2.1.7"
```
Comment (mvdnbrk) [inline package.json:21]: This can be updated to ^4.0.15.

**Review by mvdnbrk — CHANGES_REQUESTED** [review]
Dependency `laravel-mix` can be updated to ^4.0.15.

**Comment by mvdnbrk** [comment]
I have published an [updated package](https://laravel-mix.com/extensions/tailwindcss) which is compatible with TailwindCSS v1.0 and higher.

As of version 1.0 the `tailwind.config.js` is optional, so if there isn't a config file in the root of your project it uses the default Tailwind config.

If you are you using a version below 1.0 it will detect the `tailwind.js` config file and it will use that one.

**Pull request closed without merging**